src/main/pages/setup-docker/docker-cli.adoc

```diff
@@ -8,142 +8,216 @@ folder: setup-docker
 
-TARS your `/instance` into files and places them into `/backup`. These files are restoration-ready.
+|`init`                                
+|Initializes a directory with a Che install
 
-'''''
+|`offline`                            
+|Saves Che Docker images into the `TAR` files for offline installation
 
-_config_
+|`restart`                           
+|Restarts the Che services
+
+|`restore`                              
+|Restores Che configuration and data from the `/data/backup` mount
+
+|`rmi`                                  
+|Removes the Docker images for <version>, forcing a repull
+
+|`ssh <_workspace-name_> [_machine-name_]`       
+|SSH to a workspace if SSH agent enabled
+
+|`start`                                
+|Starts the Che services
+
+|`stop`                                 
+|Stops the Che services
+  
+|`sync <_workspace-name_>`                     
+|Synchronizes workspace with local directory mounted to `:/sync`
+ 
+|`test <_test-name_>`                     
+|Starts test on the Che instance
+
+|`upgrade`                             
+|Upgrades Che from one version to another with migrations and backups
+
+|`version`                             
+|Installed version and upgrade paths
+|===
+
+The following table lists the global command options that you can se with the Docker CLI image.
+
+[cols="2*", options="header"]
+|===
+|Command option
+|Description
+|`--fast`                               
+|Skips networking, version, nightly, and preflight checks
+
+|`--offline`                            
+|Runs CLI in the offline mode, loading images from the disk
+
+|`--debug`                              
+|Enable debugging of the Che server
+
+|`--trace`                              
+|Activates trace output for debugging CLI
+|===
 
-Generates a Che instance configuration thta is placed in `/instance`. This command uses puppet to generate Docker Compose configuration files to run Che and its associated server. Che’s server configuration is generated as a che.properties file that is volume mounted into the Che server when it boots. This command is executed on every `start` or `restart`.
+The CLI has the following three primary phases:
 
-If you are using a `eclipse/che:<version>` image and it does not match the version that is in `/instance/che.ver`, then the configuration will abort to prevent you from running a configuration for a different version than what is currently installed.
+* The initialization phase is executed by the `init` command. It installs version-specific files into the directory mounted to `/data`. This includes the universal configuration file named `che.env`, a version identifier, and a location where configuration files are saved. The configuration is executed by the `config` command. It takes as input, the `che.env` configuration file, the OS of your host, and then generates an OS-specific set of configuration files in the `/data/instance` directory that can be used to run an instance of Che. 
```

Comment:
The sentence beginning with "it takes as input" is awkwardly written. I'd recommend rewriting, perhaps something like this:

"It takes the `che.env` configuration file and the OS of your host system as input and generates an OS-specific set of configuration files in the `data/instance` directory."

Also, what is "it" referring to here? The `init` command or the `config` command? That should be qualified. :)

Reply:
It is the `config` command. I changed the text. Thanks :)